RNA-seq/setup/02-gastric_cancer_tximport_original.Rmd

```diff
@@ -1,5 +1,5 @@
 ---
-title: "NB cell line: gene-level summarization with `tximport`"
```

Comment:
Oh I see. Git is confused about these files and thinks they are the same but technically they are not. But I guess we don't really need both anyway?